From 4b84f243a70a8c07f6a38dad3c9411fa707f25c9 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sun, 4 Mar 2018 17:08:43 -0800
Subject: [PATCH] use _nl_msg_cat_cntr only with glibc

The musl libc provides libintl (similar to glibc)
but does not use the same internals,
so even though we are using the GNU gettext
the libintl included with the libc does not define
_nl_msg_cat_cntr and it does not need to.

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
Upstream-Status: Pending

 src/NCi18n.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/NCi18n.h b/src/NCi18n.h
index 165b0e7..bfc4ed7 100644
--- a/src/NCi18n.h
+++ b/src/NCi18n.h
@@ -59,12 +59,13 @@ inline void setTextdomain( const char * domain )
     bindtextdomain( domain,  YSettings::localeDir().c_str() );
     bind_textdomain_codeset( domain, "UTF-8" );
     textdomain( domain );
-
+#if defined(__GLIBC__)
     // Make change known
     {
 	extern int _nl_msg_cat_cntr;
 	++_nl_msg_cat_cntr;
     }
+#endif
 }
 
 
-- 
2.16.2

